### ISSUE TITLE
Post live chat message not triggering `recieve` event

### ISSUE THREAD

**Issue by danieljimeneznz** [watson-developer-cloud/assistant-web-chat-react#32]
Hi, I wasn't sure where to report this bug - but I have noticed an inconsistency in the events mechanism, specifically the [`recieve`](https://web-chat.global.assistant.watson.cloud.ibm.com/docs.html?to=api-events#receive) event does not fire when a message gets sent by the assistant when the user disconnects from an agent.

To recreate this, simply add a recieve listener to the watson instance with a service desk on the project and note the lack of this event firing after the user disconnects and the default message gets sent. I expect this event to be triggered, and don't believe this message is triggering any events listed in the api events documentation.

<img width="877" alt="image" src="https://github.com/user-attachments/assets/84935373-0f8b-483a-bf6e-eb475531b450">

**Comment by TazmanianDI** [comment]
You are correct that this is the current behavior. The `receive` event is only fired for messages that are actually sent by the assistant service and received by web chat. Messages generated internally by web chat (there's not many but the message you mention is not the only one) do not trigger the receive event.

And so, you are unfortunately correct in that there's currently no event you can use to capture this message. Do you have a specific reason to do so?
